chpl/chpl-service/src/main/java/gov/healthit/chpl/certifiedproduct/CertifiedProductDetailsManager.java

```diff
@@ -75,6 +75,13 @@ public CertifiedProductSearchDetails getCertifiedProductDetails(Long certifiedPr
         });
     }
 
+    @Transactional(readOnly = true)
+    public CertifiedProductSearchDetails getCertifiedProductDetailsSMALL(Long certifiedProductId) throws EntityRetrievalException {
```

Comment:
Could we rename this method? Maybe getCertifiedProductDetailsWithoutUnattestedCriteria ?